lib/resources/postgres.rb

```diff
@@ -79,7 +79,7 @@ def locate_data_dir_location_by_version(ver = @version)
 
       dir_list.each do |dir|
         data_dir_loc = dir if inspec.directory(dir).exist?
-        break
+        break if inspec.directory(dir).exist?
```

Comment:
I agree this needs to be a conditional, but I don't think we need to perform the same check as was performed on line 81 - that's another round-trip filesystem check.  What about

```
break if data_dir_loc
```

Reply:
I'm not a ruby guy so I had to poke around for an idiomatic version.  See next commit for a 1-line version.

Reply:
Hey, that is much prettier than what I suggested.  Thanks for your contribution!